packages/altair-electron/src/app/touchbar.ts

```diff
@@ -33,10 +40,39 @@ export class TouchbarManager {
         sendRequestButton,
         spacer,
         reloadDocsButton,
-        showDocsButton,
+        this.showDocsButton,
       ],
     });
 
     return touchBar;
   }
+
+  private removeShowDocsButton() {
+    const { items } = this.showDocsButton.parent;
+    const index = items.indexOf(this.showDocsButton);
+    if (index !== -1) {
+      items.splice(index, 1);
+    }
+  }
+
+  private addHideDocsButton() {
+    this.hideDocsButton = new TouchBarButton({
+      label: 'Hide Docs',
+      click: () => {
+        this.actionManager.hideDocs();
```

Comment:
There is no `hideDocs` action in the action manager. That needs to be added. This will not work.

Reply:
sure 
i have worked on it and added the functionality to it
by adding a new branch.